### DISCUSSION TITLE
Kimai could use a pause feature

### DISCUSSION THREAD

**Discussion by vic-t** [kimai/kimai#2365] [General]
https://github.com/kimai/kimai/issues/963

**Comment by kevinpapst** [comment]
Please vote for the pause feature request

**Comment by cryptelli** [comment]
I would also find this feature beneficial.

**Comment by WhiteTiger-IT** [comment]
I agree with the break, but it is important that marked the hour of stop / restart.
Otherwise the "End-Job" + "Start-again" options is ok.
The rounding problem is real, but in my opinion it could be solved instead by putting the default settings to the Job card with the possibility of changing it.
Therefore all Jobs will have a rounding to 30 minutes, but I can individually choose not to have rounding (or with a smaller rounding).

**Comment by alizehctl** [comment]
Yes this would be hugely beneficial. Especially for those who are working from home and get interrupted during work a lot.

**Comment by sollermun** [comment]
Has this feature been implemented? For those of us that are constantly interrupted by telephone calls, etc., it is important to be able to resume a pending timer without creating a whole new ticket.

**Comment by StefanGebhardt** [comment]
It would be helpful to have a pause field in the record. 
I do not work with the start/stop mechanism, but enter start and end times. To take into account the lunch break, for example, I have to either manipulate my start or end time or create two data records - which in turn is not wanted when listing the times to the customer later.

**Comment by markus-steffen** [comment]
For me that would be helpful too.

**Comment by dhoernel** [comment]
I vote for this feature too. I also know it from other time tracking software.

**Comment by frank42hh** [comment]
I second this. I would like to use Kimai for tracking employee work shifts in a cocktail bar. German laws require doing breaks and it would help a lot if one could add them to the shift entry instead of having to record two "half shifts" every day.

**Comment by uku69** [comment]
Would need a "pause" entry added to a time entry. So, if I start at 10:00 and work to 14:00 hrs with 30' mins break in between, I'll have one time entry start 10:00, End 14;00; Break 30' mins, duration 3:30. I don't want to use Kimai as a paermanent running tool in background, but for registring work afterwards.

**Comment by limitedAtonement** [comment]
@kevinpapst Regarding the pause feature, the clearest definition of functionality is on #963. I think that could be improved by the following:

* Every entry should have a quick-use button "start again".

I'm pretty sure this would achieve what is desired in #963 and it would make starting old tasks again very convenient. For my use case, I have several clients with various per-hour pay rates. By far the most convenient way to pick up work again from the previous day is to "start again", but requiring two clicks is somewhat burdensome, especially because I miss the first click (the ... button) so often. This is especially the case after upgrading to v2 where that button is much smaller (that's what provoked me to come here to discuss this).

I would propose something like this:

![image](https://github.com/kimai/kimai/assets/7562490/c3170fe0-3049-4e44-abd1-874aceb5b610)

I'm looking into developing this myself. The way I made that mock up image is by adding an item to the "action" buttons column and laying them side by side. Does that seem like a reasonable way to do it?

**Reply by kevinpapst** [reply]
You can can click anywhere on that row with "right click" to bring up the context menu. The "there dots" are just there to signal that there is more to explore.

I don't think that there is much more that needs to be added as input for an initial implementation, which might be added in the near future. Until them I will lock this issue.
